app/views/CreateChannelView/RoomSettings/index.tsx

```diff
@@ -22,48 +22,72 @@ export const RoomSettings = ({
 	createPrivateChannelPermission: boolean;
 	e2eEnabledDefaultPrivateRooms: boolean;
 }) => {
+	const e2eEnabled = useSetting<boolean>('E2E_Enable');
+
 	const [type, setType] = useState(true);
 	const [readOnly, setReadOnly] = useState(false);
 	const [encrypted, setEncrypted] = useState(e2eEnabledDefaultPrivateRooms);
 	const [broadcast, setBroadcast] = useState(false);
 
-	const { encryptionEnabled } = useAppSelector(state => ({
```

Comment:
By removing this block, users would be able to create encrypted channels without enabling e2ee on the app.
You're now reading new settings from the admin panel, but this is an important local one.